tools-public/toolpad/pages/waitingForMaintainer/page.yml

```diff
@@ -29,15 +29,24 @@ spec:
           - name: q
             value:
               $$jsExpression: >
-                `is:issue is:open repo:mui/${parameters.repository}
+                `is:issue repo:mui/${parameters.repository}
```

Comment:
The label on close issues is equality an issue than on open issue (a feedback we missed)